app/models/comfy/cms/page.rb

```diff
@@ -97,7 +97,7 @@ def published_at
     elsif scheduled?
       scheduled_on
     elsif ever_been_published?
-      revisions.detect { |revision| revision.data[:event].include?('published') }.created_at
+      revisions.detect { |revision| Array(revision.data[:event]).include?('published') }.created_at
```

Comment:
Don't think so. `#try?` should be calling `include?` on the object it's being called on, `revision.date[:event]` here. In any case, @asmega is probably right and this would just force this code to be bigger than it needs to be, considering the return value of `detect` is being used. Nice idea, though. :)